delivery-api-v1.0.raml

```diff
@@ -128,6 +128,9 @@ types:
       stops:
         type: Stop[]
         description: must start an element with the kind "pickup", followed by 1+ elements with the kind "destination" and followed by 0+ "fallback_destinations".
+      freezeDestinationsOrder?:
+        type: boolean
+        description: Delivery order of stops is fixed. (do not optimize it)
```

Comment:
I would add default value, please.

Reply:
I should actually remove this. I copied the current state of the private docs but this is not related to the feature and I am not sure it is released yet.

Reply:
makes sense.